src/pages/verify.tsx

```diff
@@ -39,54 +40,72 @@ export default function Verify() {
+              {"\u00A0"}Verify{"\u00A0"}
+            </Typography>
+            ?
           </Typography>
-        </Box>
 
-        <Box>
-          <Typography gutterBottom variant="h1">
-            Verify แล้วได้อะไร
-          </Typography>
-          <Typography component="div" variant="body2">
-            1. ท่านจะสามารถสร้างกิจกรรมเพื่อหาเพื่อน ๆ มาร่วมสนุกกับท่านได้
+          <Typography variant="body1" align="center" style={{ lineHeight: "40px" }}>
+            <Typography display="inline" color="primary">
+              Verify{"\u00A0"}
+            </Typography>
+            is an authentication system to provide security for you and PlayHub members who use our
+            website, as well as receive special privileges from PlayHub.
           </Typography>
-          <Typography component="div" variant="body2">
-            2. ท่านจะได้รับเครื่องหมายยืนยันตัวตนสุดเท่ในหน้าโปรไฟล์ของท่าน
+        </Stack>
+
+        <Stack spacing={2}>
+          <Typography gutterBottom variant="h2">
+            What
+            <Typography gutterBottom display="inline" variant="h2" color="primary">
+              {"\u00A0"}Verify{"\u00A0"}
+            </Typography>
+            gives you
           </Typography>
-        </Box>
+          <Box display="flex" justifyContent="center">
+            <Typography variant="body1" align="left" style={{ lineHeight: "40px" }}>
+              <Box>1. You will be able to create activity post to find friends to join you.</Box>
+              <Box>2. You will receive a cool identity badge on your profile.</Box>
+            </Typography>
+          </Box>
+        </Stack>
 
-        <Box>
-          <Typography gutterBottom variant="h1">
-            Verify อย่างไร
+        <Stack spacing={2}>
+          <Typography gutterBottom variant="h2" align="center">
+            How to
+            <Typography gutterBottom display="inline" variant="h2" color="primary">
+              {"\u00A0"}Verify{"\u00A0"}
+            </Typography>
+            ?
           </Typography>
-          <Typography component="span" variant="body2">
-            ส่ง
+
+          <Typography variant="body1" align="center" style={{ lineHeight: "40px" }}>
+            Attach
+            <Typography display="inline" variant="body1" color="error">
+              {"\u00A0"}national ID card photo{"\u00A0"}
+            </Typography>
+            to this
             <Link href={emailToAdmin} style={{ color: "blue", textDecorationLine: "underline" }}>
-              อีเมล
+              {"\u00A0"}email{"\u00A0"}
             </Link>
-            นี้ พร้อมแนบ
-            <Typography display="inline" variant="body2" color="red">
-              รูปถ่ายบัตรประจำตัวประชาชน
-            </Typography>
-            แล้วรอการตรวจสอบประมาณ 1 สัปดาห์ หากการยืนยันตัวตนสำเร็จจะมีสัญลักษณ์ &nbsp;
+            and wait for verification around 1 week. If you pass the verification, this badge
```

Comment:
Could you add ` ` (or is it `{"\u00A0"}`) after `to this` and before `and wait for ...` instead since for now the underline took part in space.

Reply:
Solved

Reply:
`{"\u00A0"}` is ` `. Solved krub, sorry ididn't recheck this page